tsconfig.json

```diff
@@ -15,5 +15,6 @@
         "strict": false
     },
     "files": [".storybook/typings.d.ts"],
-    "include": ["src/**/*.ts", "src/**/*.tsx", "examples/src/**.tsx", "types.ts", "scripts/jest/jest-setup.ts"]
+    "include": ["src/**/*.ts", "src/**/*.tsx", "examples/src/**.tsx", "types.ts", "scripts/jest/jest-setup.ts"],
+    "exclude": ["src/**/*.stories.ts", "src/**/*.stories.tsx"]
```

Comment:
why do we want to exclude stories?

Reply:
it dawned on me that components and elements in stories do not need all the required props to demonstrate specific functions.